### DISCUSSION TITLE
I can't update the plugin

### DISCUSSION THREAD

**Discussion by milevskiy2** [mattieserver/netbox-topology-views#482] [Q&A]
did this: 
```
source /opt/netbox/venv/bin/activate
install netbox-topology-views --upgrade
python3 manage.py migrate netbox_topology_views
python3 manage.py collectstatic --no-input
```

![1](https://github.com/netbox-community/netbox-topology-views/assets/167505163/28677dbc-8035-4eca-b5df-42da8c83dda1)
![2](https://github.com/netbox-community/netbox-topology-views/assets/167505163/db797416-c31e-42c7-a54e-a4920eab04c2)
![3](https://github.com/netbox-community/netbox-topology-views/assets/167505163/1e921497-530a-47b0-8f18-2d6d094fe8b6)

**Comment by dreng** [comment]
Hi,

it seems that you've found a solution yourself. Since it may help other users, please be so kind and tell us what you did to solve your problem.

**Reply by milevskiy2** [reply]
HI!



![4](https://github.com/netbox-community/netbox-topology-views/assets/167505163/3af49d26-1fd2-4db9-91d7-d2303d5591bf)


```
sudo -s
source /opt/netbox/venv/bin/activate
cd /opt/netbox/netbox
pip uninstall netbox-topology-views
pip install netbox-topology-views
python3 manage.py migrate netbox_topology_views
python3 manage.py collectstatic --no-input
```

`systemctl restart netbox`
